src/components/Form/Inputs/SearchSelect/SearchSelect.tsx

```diff
@@ -85,11 +87,11 @@ export const SearchSelect = (props: IProps) => {
         }
 
         return (
-          value.value
+          `${value?.value}`
```

Comment:
This could generate an error since the operator? returns false if it does not exist.
could generate something like 
`'false'.toString ().toLowerCase () .toLowerCase ()`